src/core/meshcop/joiner_router.cpp

```diff
@@ -232,18 +232,15 @@ void JoinerRouter::HandleRelayTransmit(Coap::Message &aMessage, const Ip6::Messa
     }
 }
 
-otError JoinerRouter::DelaySendingJoinerEntrust(const Ip6::MessageInfo &aMessageInfo, const uint8_t *aKek)
+Coap::Message *JoinerRouter::PrepareJoinerEntrustMessage(void)
```

Comment:
nit: how about moving `PrepareJoinerEntrustMessage()` closer to `SendJoinerEntrust()` where it is used?

Reply:
Sounds good. Kept it here to minimize the git diff. Will move it.